examples/feature_selection/plot_select_from_model_diabetes.py

```diff
@@ -143,3 +143,67 @@
 # attribute. The forward SFS is faster than the backward SFS because it only
 # needs to perform `n_features_to_select = 2` iterations, while the backward
 # SFS needs to perform `n_features - n_features_to_select = 8` iterations.
+
+# %%
+# Selecting Features Using Sequential Feature Selection and Negative tol Values
```

Comment:
It looks like the other headlines are capitalised like this:
```suggestion
# Selecting features using Sequential Feature Selection and negative tolerance values
```

Also extended the tol jargon to "tolerance".